woocommerce/woocommerce-gutenberg-products-block
# Increase contrast of checkout error messages when using dark mode

Currently, the color contrast of the form labels and the error messages in the checkout block, when using the dark mode, is insufficient, when an error occurs.
For now, a change was made to lighten the $alert-red color by 30% to achieve a better contrast ratio based on @nielslange recommendation.
This issue may actually be an entry point for a full array of contrast problems we may have, even outside of the cart & checkout blocks.


Fixes #5740 

#### Accessibility


#### Other Checks


### Screenshots

Before:
![image](https://user-images.githubusercontent.com/17236129/164450118-2652edf1-7b16-47c0-8d7c-df3a3f5f3404.png)


Wip solution:
![image](https://user-images.githubusercontent.com/17236129/164450213-051fd1f8-274c-46e3-9018-61fca6bc2d7b.png)


### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Use a theme that supports dark mode and enable it, e.g. [Eksell](https://wordpress.org/themes/eksell/).
2. Create a test page and add the checkout block.
3. Click on the checkout block and activate the option Style » Dark mode inputs in the sidebar.
4. Save that test page and go to the frontend.
5. Add a product to the cart and go to the test page with the checkout block.
6. Clear all form fields, if they're pre-filled, and click on place order.
7. See the error messages with sufficient color contrast.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).



### Performance Impact



### Changelog

> Contrast improvement for checkout error messages when displayed over a theme's dark mode.
